### ISSUE TITLE
Gradle deprecated property warning

### ISSUE THREAD

**Issue by asaarnak** [aws-amplify/amplify-flutter#2417]
The Report.enabled property has been deprecated. This is scheduled to be removed in Gradle 8.0. Please use the required property instead. See https://docs.gradle.org/7.6/dsl/org.gradle.api.reporting.Report.html#org.gradle.api.reporting.Report:enabled for more details.
        at coverage_2qtuie4wcj50cut782zsfrn4r$_run_closure3$_closure4$_closure5$_closure6.doCall(/home/user/.pub-cache/hosted/pub.dartlang.org/amplify_auth_cognito_android-0.6.10/android/coverage.gradle:32)

https://github.com/aws-amplify/amplify-flutter/blob/main/packages/auth/amplify_auth_cognito_android/android/coverage.gradle#L32-L34

https://docs.gradle.org/7.6/dsl/org.gradle.api.reporting.Report.html#org.gradle.api.reporting.Report:enabled

**Comment by asaarnak** [comment]
@dnys1
This was fixed on `master` branch.
I noticed the same issue on `next` branch.
amplify_auth_cognito_android-1.0.0-next.2/android/coverage.gradle:32
amplify_flutter_android-1.0.0-next.2/android/coverage.gradle:32

https://docs.gradle.org/7.6/dsl/org.gradle.api.reporting.Report.html#org.gradle.api.reporting.Report:enabled
```
The Report.enabled property has been deprecated. |  
-- | --
  | This is scheduled to be removed in Gradle 8.0. |  
  | Please use the required property instead.
```

**Comment by dnys1** [comment]
Good catch, @asaarnak! I've opened https://github.com/aws-amplify/amplify-flutter/pull/2553 to handle that in `next` as well.
